src/build-tx/transaction.js

```diff
@@ -45,7 +45,7 @@ export type Input = {
 
 export type Transaction = {
     inputs: Array<Input>,
-    outputs: Permutation<Output>, // not in trezor.js, but needed for metadata saving
+    outputs: Permutation<Output> | { sorted: Array<Output>, _permutation?: typeof undefined }, // not in trezor.js, but needed for metadata saving
```

Comment:
I'm wondering if we really need this special type that exposes implementation details of the `Permutation` class. Can't we handle the case where `skipPermutation === true` by returning permutation that is an identity. i.e. `i === j` for all elements?

Reply:
f1f60fb7ee11f098739da5f79d3d88e99fe842db

Reply:
Thanks!